src/pug/characters/aoba.pug

```diff
@@ -6,16 +6,19 @@ include ../templates
   title: {
     image: "https://vignette.wikia.nocookie.net/new-game/images/f/f2/Th-c1.png",
     name: "Aoba",
-    quote: '"A funny quote from me"'
+    quote: '"Ganbaruzoi!"'
   },
   strengths: [
     "Is adorable",
-    "Is a legal loli",
-    "Can do the art thing",
-    "Has occasional sadism"
+    "Doesn't drink nor smoke",
+    "Very good at art",
+    "Has a bear sleeping bag",
+    "Should absolutley go out with Hifumi, they're such a cute couple"
```

Comment:
still wrong, also this isn't a strength

Reply:
w-which one? the not drinking or smoking or the not going out with Hifumi?

Reply:
absolutley

Reply:
```suggestion
-    "Should absolutley go out with Hifumi, they're such a cute couple"
```

Reply:
```suggestion
    
```